.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+# Description
+A clear and concise description of what the feature or bug fix is intended to do.
+
+# Implementation Strategy
+A clear and concise description of the implementation strategy.
+
+# Pull Request Acceptance Criteria
+The following pull request review steps must be completed before merging:
+
+ - [ ] Review code for style, design, and accuracy
+ - [ ] Review unit tests for completeness and accuracy
+ - [ ] Verify ``changelog.rst`` entry
+ - [ ] Verify Github CI/CD actions passing
+ - [ ] Verify external WLM tests passing
+ - [ ] Review documentation for proper content, build, and rendering
+ - [ ] Review and run examples for sufficient breadth and accuracy
+ - [ ] Independent developer feature stress test
+ - [ ] Prior to closing the associated issue, verify documentation and examples have been properly uploaded to ``craylabs.org`` and all CI/CD tests pass post merge
```

Comment:
Feels potentially misplaced in process. I don't think a reviewer will come back after the merge into development and verify doc uploads - Should this be captured in developer steps?